tests/macros/utils.test.js

```diff
@@ -26,13 +28,31 @@ const ERROR_TEST_CASES = [
     }
 ];
 
-describe('test lintHTML function', function() {
-    for (const test of ERROR_TEST_CASES) {
-        it(test.title, function() {
-            expect(lintHTML(test.html)).toContain(test.error);
+describe('Macro test utils', function() {
+    describe('test lintHTML function', function() {
+        for (const test of ERROR_TEST_CASES) {
+            it(test.title, function() {
+                expect(lintHTML(test.html)).toContain(test.error);
+            });
+        }
+        it('with valid HTML input', function() {
+            expect(lintHTML('<div>This is nice</div>')).toBeFalsy();
         });
-    }
-    it('with valid HTML input', function() {
-        expect(lintHTML('<div>This is nice</div>')).toBeFalsy();
+    });
+
+    it('`readFixture` works', function() {
+        let result = String(readFixture('utils-test.txt'));
```

Comment:
I think it would be nice if the fixture files were in a utils-test/ subdirectory, to keep the fixtures directory a bit cleaner. And that would exercise the case where there is more than one argument to readFixture.  Maybe utils-test/fixture.txt and utils-test/fixture.json or something like that?